ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -5,6 +5,9 @@
 # If a device is set to "is_deployed: false" at the host_vars level, then shutdown interfaces towards undeployed peers.
 shutdown_interfaces_towards_undeployed_peers: true
 
+# If a device is set to "is_deployed: false" at the host_vars level, then shutdown bgp towards undeployed peers.
+shutdown_bgp_towards_undeployed_peers: true
```

Comment:
we might need the same key for overlay. @Shivani-gslab

Reply:
This is a global key for both overlay and underlay